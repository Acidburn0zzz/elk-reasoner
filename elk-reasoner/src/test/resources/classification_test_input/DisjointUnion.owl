Prefix(:=<http://example.org/> )
Prefix(owl:=<http://www.w3.org/2002/07/owl#> )
Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#> )

Ontology(

Annotation(rdfs:label "DisjointUnion causes inconsistency of all classes")

DisjointUnion(:A :B :C)
SubClassOf(:A :B)
SubClassOf(:A :C)

)