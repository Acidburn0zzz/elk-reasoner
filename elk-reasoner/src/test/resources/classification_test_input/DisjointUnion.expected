Prefix(:=<http://example.org/> )
Prefix(owl:=<http://www.w3.org/2002/07/owl#> )
Ontology(

EquivalentClasses(:A owl:Nothing)
EquivalentClasses(:B owl:Nothing)
EquivalentClasses(:C owl:Nothing)

)